src/components/Homepage/Intro.js

```diff
@@ -37,6 +37,8 @@ const IntroRow = styled(Row)`
 `
 
 const StyledCardTitle = styled(CardTitle)`
+  font-weight: 400;
```

Comment:
Looks like this is in `CardTitle` as `is('regular')`?

Reply:
Kinda weird that an override is called "regular" :see_no_evil:

Reply:
oh boy

Reply:
So shall we use `regular` here instead of repeating the 400 constant?